src/development/python_package_release_procedure.rst

```diff
@@ -0,0 +1,117 @@
+.. _Python_packaging: https://packaging.python.org/tutorials/packaging-projects/
+
+Versioning Procedure
+--------------------
+
+As part of our goal to align all developmental efforts to one standard, we
+have documented a procedure of how we would like all the *SKA* developer to 
+version their releases and what process to follow in ensuring that they are 
+able to make use of the existing Gitlab CI/CD pipeline to automate the
+python (for now) packages and have them published on the *SKA* *Nexus* registry.
+
+
+Versioning scheme to use
+========================
+
+The scheme chosen to be adopted by the *SKA* developer community is the semantic versioning
+scheme.
+More information regarding this scheme can be found on the Python_packaging_ site.
+
+How to mark a release
+=====================
+
+For this, a developer should make use of the git tags to indicate that this 
+current commit is to be released.
+For example:
+
+.. code:: bash
+
+  $ git tag -a "1.0.0 -m "Release 1.0.0."
```

Comment:
As per Marco's comment in the packaging doc, we need to have a better example of the commit message. Please see the ST-150 doc for an example.